pallets/bootstrap/src/mock.rs

```diff
@@ -273,19 +273,21 @@ mockall::mock! {
 			amount: Balance,
 		) -> DispatchResult;
 
-		fn promote_pool(liquidity_token_id: TokenId) -> bool;
+		fn update_pool_promotion(liquidity_token_id: TokenId, liquidity_mining_issuance_percent: Option<Percent>);
```

Comment:
I see that you used percent here, but we agreed that the reward would be based on ratio and not a percentage value.

Reply:
Nope, I remember that we discussed it to be Percent, over a call I think.
I can realign it, but it'll take some time.

Reply:
We agreed on this with Gleb a while ago, and it's also in the ticket description MGX-115. With this implementation, we would need to do a council proposal for every promoted pool update as we would need to modify every pool to match % we need as they will probably always be divided into 100% ... we will need to rework this 😬

Reply:
"The weights should be in ratio..."
^This is the first time I am seeing this.
The product spec mentioned "weight or percent".

"we would need to do a council proposal for every promoted pool update as we would need to modify every pool to match % we need as they will probably always be divided into 100%"
This is untrue. This is exactly the reason it was made underflow and overflow resilient.

Reply:
Ok, probably some miscommunication happened. Glad it was easy to fix. Apologies for reviewing this too late.